app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+class Animal:
+    health = 100
+    hidden = False
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.alive.append(self)
+
+    def __repr__(self) -> str:
+        for animal in self.alive:
+            return "{" \
+                   + f"Name: {self.name}, " \
+                     f"Health: {self.health}, " \
+                     f"Hidden: {self.hidden}" \
+                   + "}"
+
+
+class Herbivore(Animal):
+    def hide(self) -> None:
+        self.hidden = not self.hidden
+
+
+class Carnivore(Animal):
+    @staticmethod
+    def bite(other: "Animal") -> None:
+        if other.hidden is not True \
```

Comment:
is not True are not needed in both cases. other.hidden and isinstance(other, Carnivore) return booleans, you can change them to not precede them